.gitignore

```diff
@@ -1,7 +1,9 @@
 node_modules
-legit-tests.js
-dom.js
-middleware.js
-middleware
-no-dom.js
-tests.js
+lib
```

Comment:
It does out to the root directory. When I switched branched, old `lib` folder stayed there and it was removed from `gitignore`  So, I added it gitignore instead of deleting those lib files. I'll remove that line.